### DISCUSSION TITLE
[Feature] Add a filter captions translation/customization/filter row?

### DISCUSSION THREAD

**Discussion by ThisIsRex** [revolist/revogrid#653] [General]
It would be a good idea to allow user to translate filter names, show filter row below column headers and create custom filter popup renderer, like that:
```js
{
    //...
    filterRenderer: (h, filters, applyFn, cancelFn) {
        return (
            <div... >...</div>
        )
    }
}
```

**Comment by van-huyen** [comment]
I think you can customize filter like this:
https://revolist.github.io/revogrid/guide/filters.html#advance-usage
![image](https://user-images.githubusercontent.com/83264685/138982116-1cc6bb76-2ebe-4208-b9c5-fc8f383f77d9.png)

**Comment by ThisIsRex** [comment]
@van-huyen yes, but you can't customize stock filter names and button captions
